modules/mono/glue/GodotSharp/GodotSharp/Core/Vector2.cs

```diff
@@ -580,9 +583,11 @@ internal void Normalize()
         }
 
         /// <summary>
-        /// Returns this vector slid along a plane defined by the given <paramref name="normal"/>.
+        /// Returns a new vector resulting from sliding this vector along a line with normal <paramref name="normal"/>.
+        /// The resulting new vector is perpendicular to <paramref name="normal"/>, and is equivalent to this vector minus its projection on <paramref name="normal"/>.
+        /// See also <see cref="Project(Vector2)"/>.
         /// </summary>
-        /// <param name="normal">The normal vector defining the plane to slide on.</param>
+        /// <param name="normal">The normal vector of the plane to slide on.</param>
         /// <returns>The slid vector.</returns>
         public readonly Vector2 Slide(Vector2 normal)
```

Comment:
The C# documentation is missing the note about the vector `normal` having to be normalized. Does that not apply to C#?

Reply:
I did slip my mind, and the code not having the check didn't help.